### ISSUE TITLE
Feature Request - Set User Volume in Server XML Config for TeamTalk5 Server

### ISSUE THREAD

**Issue by AgentSmith2001** [BearWare/TeamTalk5#1820]
### Description

The current version of the TeamTalk5 server does not have an option for admins to set user volume levels in the server XML configuration file. This feature request is to add a setting that allows admins to set a default volume level for individual users when they log into the server.
The proposed feature would provide two significant benefits. Firstly, it would prevent users from raising their volume levels beyond what the admin permits, thus avoiding loud and disruptive noises. Secondly, it would benefit those using Samsung devices with low volume levels even at 100% gain by allowing them to have a more acceptable volume level.
This feature request is similar to the "Fixed volume for all users" setting available in the channel settings. However, the proposed feature provides more granular control over individual user volume levels.
Implementing this feature would provide admins with greater control and customization options, thereby improving the overall user experience. Users logging into the server would have a consistent and appropriate volume level, preventing any unpleasant surprises or disruptions.
Thank you for considering this feature request.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by amirmahdifard** [comment]
yeah i agree. There should be more user rigths for admins to controle users, including preventing them from sending private messages to people and sending channel messages that i've requested before and developers just closed it
